### ISSUE TITLE
Class name is being duplicated using jekyll plugin

### ISSUE THREAD

**Issue by xt0rted** [primer/octicons#334]
I'm not 100% sure where this issue is originating, but I forked [github/personal-website](https://github.com/github/personal-website) and the repo icons in the My Projects section are being emitted with `octicon octicon-repo` multiple times as you get farther down the list.

The first occurrence has `octicon octicon-repo` once, but the 9th occurance has `octicon octicon-repo octicon octicon-repo octicon octicon-repo octicon octicon-repo octicon octicon-repo octicon octicon-repo octicon octicon-repo octicon octicon-repo octicon octicon-repo mr-1 v-align-middle`.

I couldn't find an example of this on github.com but you can see it on my test site at https://xt0rted.github.io/personal-website/ and the line in the template where this is being added is at [_includes/repo-card.html#L6](https://github.com/github/personal-website/blob/746e8c2bb12135a61644cc808c238ddff9501e7b/_includes/repo-card.html#L6).

**Comment by mdo** [comment]
Ran into this today myself. It seems the repetition might be attached to my use of the Octicons in an include? Using it inside a page, I don't get classes repeated, but in my sidebar I got this.

`<svg height="24" class="octicon octicon-home octicon octicon-home octicon octicon-home octicon octicon-home octicon octicon-home octicon octicon-home octicon octicon-home octicon octicon-home octicon octicon-home octicon octicon-home octicon octicon-home octicon octicon-home octicon octicon-home octicon octicon-home octicon octicon-home octicon octicon-home octicon octicon-home octicon octicon-home octicon octicon-home octicon octicon-home octicon octicon-home octicon octicon-home octicon octicon-home octicon octicon-home octicon octicon-home octicon octicon-home octicon octicon-home octicon octicon-home octicon octicon-home octicon octicon-home octicon octicon-home octicon octicon-home octicon octicon-home octicon octicon-home octicon octicon-home octicon octicon-home octicon octicon-home octicon octicon-home octicon octicon-home octicon octicon-home octicon octicon-home octicon octicon-home octicon octicon-home octicon octicon-home octicon octicon-home octicon octicon-home octicon octicon-home octicon octicon-home octicon octicon-home octicon octicon-home octicon octicon-home octicon octicon-home octicon octicon-home octicon octicon-home octicon octicon-home octicon octicon-home octicon octicon-home octicon octicon-home octicon octicon-home octicon octicon-home octicon octicon-home octicon octicon-home octicon octicon-home octicon octicon-home octicon octicon-home octicon octicon-home octicon octicon-home octicon octicon-home octicon octicon-home octicon octicon-home octicon octicon-home octicon octicon-home octicon octicon-home octicon octicon-home octicon octicon-home octicon octicon-home octicon octicon-home octicon octicon-home octicon octicon-home octicon octicon-home octicon octicon-home octicon octicon-home octicon octicon-home octicon octicon-home octicon octicon-home octicon octicon-home octicon octicon-home octicon octicon-home octicon octicon-home octicon octicon-home octicon octicon-home octicon octicon-home octicon octicon-home octicon octicon-home octicon octicon-home octicon octicon-home octicon octicon-home octicon octicon-home octicon octicon-home octicon octicon-home octicon octicon-home octicon octicon-home octicon octicon-home octicon octicon-home octicon octicon-home octicon octicon-home octicon octicon-home octicon octicon-home octicon octicon-home octicon octicon-home octicon octicon-home octicon octicon-home octicon octicon-home octicon octicon-home octicon octicon-home octicon octicon-home octicon octicon-home octicon octicon-home octicon octicon-home octicon octicon-home octicon octicon-home octicon octicon-home octicon octicon-home octicon octicon-home octicon octicon-home octicon octicon-home octicon octicon-home octicon octicon-home octicon octicon-home octicon octicon-home octicon octicon-home octicon octicon-home octicon octicon-home octicon octicon-home octicon octicon-home octicon octicon-home octicon octicon-home octicon octicon-home octicon octicon-home octicon octicon-home octicon octicon-home octicon octicon-home octicon octicon-home octicon octicon-home octicon octicon-home octicon octicon-home octicon octicon-home octicon octicon-home octicon octicon-home octicon octicon-home octicon octicon-home ." viewBox="0 0 24 24" version="1.1" width="24" aria-hidden="true"><path fill-rule="evenodd" d="M11.03 2.59a1.5 1.5 0 011.94 0l7.5 6.363a1.5 1.5 0 01.53 1.144V19.5a1.5 1.5 0 01-1.5 1.5h-5.75a.75.75 0 01-.75-.75V14h-2v6.25a.75.75 0 01-.75.75H4.5A1.5 1.5 0 013 19.5v-9.403c0-.44.194-.859.53-1.144l7.5-6.363zM12 3.734l-7.5 6.363V19.5h5v-6.25a.75.75 0 01.75-.75h3.5a.75.75 0 01.75.75v6.25h5v-9.403L12 3.734z"></path></svg>`
